src/textual/pilot.py

```diff
@@ -1,12 +1,36 @@
 from __future__ import annotations
 
 import asyncio
-from typing import Generic
+from typing import Any, Generic
 
 import rich.repr
 
 from ._wait import wait_for_idle
 from .app import App, ReturnType
+from .css.query import QueryType
+from .events import Click, MouseDown, MouseMove, MouseUp
+from .geometry import Offset
+from .widget import Widget
+
+
+def _get_mouse_message_arguments(
+    target: Widget, offset: Offset = Offset(), button: int = 0
+) -> dict[str, Any]:
+    x, y = offset
+    click_x, click_y, _, _ = target.region.translate(offset)
+    message_arguments = {
```

Comment:
Would there be any tangible benefit in a TypedDict here?

Reply:
I don't think it'd be worth it personally given the limited usage of this dict.